src/contracts/interfaces/IDelegationManager.sol

```diff
@@ -334,10 +334,37 @@ interface IDelegationManager is ISignatureUtils, IDelegationManagerErrors, IDele
     ) external returns (bytes32[] memory);
 
     /**
-     * @notice Used to complete the specified `withdrawal`. The caller must match `withdrawal.withdrawer`
-     * Withdrawals remain slashable during the withdrawal delay period and the actual withdrawn shares are calculated
-     * based off the scaledShares.
-     * @param withdrawal The Withdrawal to complete.
+     * @notice Used to complete the lastest queued withdrawal.
+     * @param tokens Array in which the i-th entry specifies the `token` input to the 'withdraw' function of the i-th Strategy in the `withdrawal.strategies` array.
+     * @param receiveAsTokens If true, the shares calculated to be withdrawn will be withdrawn from the specified strategies themselves
+     * and sent to the caller, through calls to `withdrawal.strategies[i].withdraw`. If false, then the shares in the specified strategies
+     * will simply be transferred to the caller directly.
+     * @dev beaconChainETHStrategy shares are non-transferrable, so if `receiveAsTokens = false` and `withdrawal.withdrawer != withdrawal.staker`, note that
+     * any beaconChainETHStrategy shares in the `withdrawal` will be _returned to the staker_, rather than transferred to the withdrawer, unlike shares in
+     * any other strategies, which will be transferred to the withdrawer.
+     */
+    function completeQueuedWithdrawal(
```

Comment:
So all legacy withdrawals need to be completed with the old interface if they aren't queued with enumerable sets and new mappings. We should make it more clear here in the interface and on the view functions like `getQueuedWithdrawals` below

Reply:
I'm wondering if we should have a combined interface so that stakers can complete withdrawals of both legacy and post-slashing withdrawals in one call. At the moment, legacy withdrawals have to be completed with the old interface.

Reply:
They can use the old interface to complete both types of withdrawals